src/components/AVPlayer/AVPlayer.js

```diff
@@ -7,7 +7,7 @@ import { Player, utils, withMediaProps } from 'react-media-player';
 import enableInlineVideo from 'iphone-inline-video';
 import { withNamespaces } from 'react-i18next';
 import classNames from 'classnames';
-import { Icon } from 'semantic-ui-react';
+import {Button, Icon} from 'semantic-ui-react';
```

Comment:
1. Create an enum for "start"
2. Transfer function for handleMuteUnmute to AVMuteUnmute. Use:
        this.props.onMuteUnmute && this.props.onMuteUnmute();
3. Make handlers in AVMuteUnmute to be one-liners